src/dotnet-grpc/Options/CommonOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.CommandLine;
+using System.IO;
+using Grpc.Dotnet.Cli.Properties;
+
+namespace Grpc.Dotnet.Cli.Options
+{
+    internal static class CommonOptions
+    {
+        public static Option ProjectOption() =>
+            new Option(
+                aliases: new[] { "-p", "--project" },
+                description: CoreStrings.ProjectOptionDescription,
+                argument: new Argument<FileInfo> { Name = "project" });
+
+        public static Option ServiceOption() =>
+            new Option(
+                aliases: new[] { "-s", "--services" },
+                description: CoreStrings.ServiceOptionDescription,
+                argument: new Argument<Services> { Name = "services" });
+
+        public static Option AccessOption() =>
+            new Option(
+                aliases: new[] { "--access" },
+                description: CoreStrings.AccessOptionDescription,
+                argument: new Argument<Access> { Name = "access" });
+
+        public static Option AdditionalImportDirsOption() =>
+            new Option(
+                aliases: new[] { "-a", "--additional-import-dirs" },
```

Comment:
IMHO the short name of the flag isn't self-descriptive enough.
protoc uses `-I` as short form of `--proto-path=` because that's a de-facto standard flag name for includes in the compiler world, while `-a` reminds me of `--all` at best. WDYT?

The long name of the flag is fine because "AdditionalImportDirs is already provided by Grpc.Tools

Reply:
I'll use -i (lowercase) since that seems more standard.